mmdet/datasets/samplers/class_aware_sampler.py

```diff
@@ -58,7 +59,8 @@ def __init__(self,
         # Must be the same across all workers. If None, will use a
         # random seed shared among workers
         # (require synchronization among all workers)
-        self.seed = sync_random_seed(seed)
+        device = get_device()
+        self.seed = sync_random_seed(seed, device)
```

Comment:
can we simply call get_device inside the function of sync_random_seed? Thus, we do not need to modify the code of samplers and only need to update the function sync_random_seed

Reply:
We can use the proposed logic.